cfme/common/__init__.py

```diff
@@ -431,8 +431,9 @@ def add_tag(self, item_objects, tag=None, cancel=False, reset=False):
         """
         view = navigate_to(self, 'All')
         for item in item_objects:
-            name = item if isinstance(item, six.string_types) else item.name
-            view.entities.get_entity(surf_pages=True, name=name).check()
+            kwargs = {keyword_name: item if isinstance(
```

Comment:
splitting this ternary makes it harder to read, please play with these line breaks a bit to get this to be a bit more clear.

Reply:
@mshriver  Is it better that way?
```python
kwargs = {keyword_name: item if isinstance(item, six.string_types) else
getattr(item, keyword_name)}
```